integration-libs/digital-payments/src/checkout/store/actions/dp-checkout-payment-details.action.ts

```diff
@@ -0,0 +1,50 @@
+import { DP_CHECKOUT_PAYMENT_DETAILS } from '../digital-payments-state';
```

Comment:
For new features we stopped using ngrx as a way to cache responses and invoke API calls.

Instead queries and commands should be used. Example of feature leveraging that is new user library.

From what I've seen in this PR this would make the whole integration library much smaller, as almost half of it is just ngrx boilerplate.